conda.yml

```diff
@@ -1,9 +1,15 @@
 name: gt4sd
+channels:
+  - https://conda.anaconda.org/pyg
+  - https://conda.anaconda.org/pytorch
 dependencies:
   - python>=3.7,<3.8
   - pip>=19.1,<20.3
+  - pytorch
```

Comment:
not a fan but if we have to keep it I would provide a lower bound like we do in requirements.txt

Reply:
not a fan at all either. If you have a better solution, I'd be more than happy to implement it.

Reply:
I set lower bounds to pytorch and pytorch-scatter now